feature/experimental/isis/otg_tests/lsp_updates_test/lsp_updates_test.go

```diff
@@ -38,12 +38,22 @@ func TestMain(m *testing.M) {
 
 func TestOverloadBit(t *testing.T) {
 	ts := session.MustNew(t).WithISIS()
+	// Only push DUT config - no adjacency established yet
+	if err := ts.PushDUT(context.Background(), t); err != nil {
+		t.Fatalf("Unable to push initial DUT config: %v", err)
+	}
+	isisPath := session.ISISPath(ts.DUT)
+	overloads := isisPath.Level(2).SystemLevelCounters().DatabaseOverloads()
+	//Lookup the initial value for 'database-overloads' leaf counter after config is pushed to DUT & before adjacency is formed
+	getDbOlInitCount := gnmi.Lookup(t, ts.DUT, overloads.State())
+	olVal, present := getDbOlInitCount.Val()
+	if !present {
+		olVal = uint32(0)
```

Comment:
Need to use `olVal + 1` as the value to check against in line 80: https://github.com/openconfig/featureprofiles/pull/2118/files#diff-ab159331c14173462166d7fe0a01f6364545fe11fcae28f6892fae8286b8ab65R80

Reply:
updated